app/models/user.rb

```diff
@@ -1,12 +1,12 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
+  devise :confirmable, :database_authenticatable, :registerable,
```

Comment:
- You have done great so far :+1: :100: But there are a few changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] Looking at the video below, you can see that the migrations failed when I ran `rails db:migrate` with the error of `relation "users" does not exist`. This is because rails runs the migrations from top to down. I.e, the migration files at the top runs first followed by others sequentially. Looking at you migration folder, I can see that your migrations are not placed in the correct order. It is supposed to be `users` first, then `posts`, followed by `comments`, and then `likes`. You can rectify these by generating the migrations again or take a rather different approach of changing the timestamp to match the correct order. :+1: :+1: :100: I'll suggest you always remember this in your future projects to avoid deployment errors. :wave: :+1:  

- - Screenshot:
![Screenshot from 2023-10-19 15-38-56](https://github.com/ctokoli/Blog/assets/107632780/9c4e5756-1573-464c-9831-d08bd57fe8e0)

- - Video reference:

https://github.com/ctokoli/Blog/assets/107632780/feaa8212-d6f9-45fa-a078-20e07db969aa